java/kotlin-extractor/src/main/kotlin/KotlinFileExtractor.kt

```diff
@@ -826,10 +826,13 @@ open class KotlinFileExtractor(
                 }
 
                 if (bf != null && extractBackingField) {
-                    val fieldId = extractField(bf, parentId)
-                    tw.writeKtPropertyBackingFields(id, fieldId)
-                    if (p.isDelegated) {
-                        tw.writeKtPropertyDelegates(id, fieldId)
+                    val fieldParentId = useDeclarationParent(getFieldParent(bf), false)
+                    if (fieldParentId != null) {
```

Comment:
Should we give an error in the null case?

Reply:
We already get one from `useDeclarationParent`